roadmap/implementors-guide/guide.md

```diff
@@ -937,13 +937,16 @@ If the candidate was not made available on another fork of the relay chain, the
 
 If the availability process passes, the remote dispute is ready to be included on this chain. As with the local dispute, validators self-select based on a VRF. Given that a remote dispute is likely to be replayed across multiple forks, it is important to choose a VRF in a way that all forks processing the remote dispute will have the same one. Choosing the VRF is important as it should not allow an adversary to have control over who will be selected as a secondary approval checker.
 
-Some choices for VRF:
-  - Session-start VRF? xor candidate hash?
-  - Last session-start VRF?
-  - use VRF from relay-chain block on another fork where the parablock was included. There are likely multiple choices for this.
-
 After enough validator self-select, under the same escalation rules as for local disputes, the Remote dispute will conclude, slashing all those on the wrong side of the dispute. After concluding, the remote dispute remains open for a set amount of blocks to accept any further proof of additional validators being on the wrong side.
 
+### Slashing and Incentivization
+
+The goal of the dispute is to garner a 2/3+ (2f + 1) supermajority either in favor of or against the candidate.
+
+For remote disputes, it is possible that the parablock disputed has never actually passed any availability process on any chain. In this case, validators will not be able to obtain the PoV of the parablock and there will be relatively few votes. We want to disincentivize voters claiming validity of the block from preventing it from becoming available, so we charge them a small distraction fee for wasting the others' time if the dispute does not garner a 2/3+ supermajority on either side. This fee can take the form of a small slash or a reduction in rewards.
```

Comment:
".. a small slash that does not chill or .."

We should discuss if the distraction fee should chill/boot validators @AlistairStewart 

Pro-chill: Anyone who voted valid could gossip the pieces themselves. 
Anti-chill: Attackers could file false disputes so that DoS attacks cause chilling.  

Can DoS attack chill anyways via ImOnline?  If so, maybe fix that one day..

Reply:
Pro-chill: currently only nonzero slash chills.

Reply:
@burdges What do you say about deferring this to a follow-up PR? I don't believe it substantially alters the meaning of the text, but is worth addressing

Reply:
Sure